apps/api/src/app/blueprint/usecases/get-blueprint/get-blueprint.command.ts

```diff
@@ -1,8 +1,8 @@
-import { IsDefined, IsMongoId } from 'class-validator';
+import { IsDefined, IsString } from 'class-validator';
 import { BaseCommand } from '@novu/application-generic';
 
 export class GetBlueprintCommand extends BaseCommand {
   @IsDefined()
-  @IsMongoId()
+  @IsString()
```

Comment:
The thing that I was not sure about was if we needed to change this param and where to provide the data that the API user can pass either id or trigger identifier.

Reply:
i am ok with templateIdOrIdentifier :)